src/components/updatemenus/attributes.js

```diff
@@ -57,6 +57,37 @@ module.exports = {
         ].join(' ')
     },
 
+    openreverse: {
```

Comment:
Ahh… maybe orientation should encode both? The problem is that we have *two* orientations but *four* directions. What about `orientation: 'l'|'r'|'u'|'d'`